api/api_helpers.py

```diff
@@ -0,0 +1,47 @@
+"""File holding some reusable methods used in the API addon submission tests"""
+
+
+def verify_addon_response_details(payload, response, request):
```

Comment:
The main problem here is that the request JSON object required for the API request to be valid (which I'm storing in the `payloads` file https://github.com/mozilla/addons-release-tests/pull/560/files#diff-f0ef8dacf8bc81e824c8da961af503cc69fecfac3b9405eb6e32d64afc92ec41R4-R70 doesn't have the same exact data structure as the response JSON object returned by API. The response has more nesting and some field keys differ from the request keys. This is why the method I made is more complicated, because I need to accout for those variations (which is also the reason why at first I've opted for simple assertions to make sure I'm comparing all the necessary values).

Reply:
But it seems you still do a comparison...so I still think a dataclass here is better than putting everything into a list. You just create one dataclass and fill it with the ```payload`` values and then parse the ```response``` values into that class and do the compare.

Reply:
If you want to run these and file a ticket to do the dataclass upgrade I'm okay with that, but please link the ticket before you merge.